package com.jyz.study.jdk.exception;

import junit.framework.TestCase;

/**
 * 
 * @author JoyoungZhang@gmail.com
 *
 */
public class ExceptionTest extends TestCase{

    public void test(){
	System.out.println(123);
    }
    
}
